client/app/pod/popular.service.js

```diff
@@ -0,0 +1,39 @@
+angular.module('throwcast.popular')
+
+.factory('popService', function ($http) {
+  var data = {};
+  return {
+    getPodcast: function () {
+      return $http.get('http://localhost:8888/api/podcasts/').then( function (res) {
+        data.podcasts = res.data.data;
+      });
+    },
+
+    getStations: function () {
+      return $http.get('http://localhost:8888/api/stations/').then( function (res) {
+        data.stations = res.data.data;
+      });
+    },
+
+    getStationPodcast: function (station, index) {
+      return $http.get('http://localhost:8888/api/stations/' + station._id + '/podcasts/').then(function (stationPodcasts) {
+        data.selectedStationPodcasts = stationPodcasts.data;
+        data.selected = index;
+      });
+    },
+
+    play: function (link) {
```

Comment:
I don't think this belongs in the podcast service. This probably belongs in a player service. Maybe leave it in the controller until you create that service.